inst/include/population_dynamics/fleet/fleet.hpp

```diff
@@ -203,14 +203,15 @@ struct Fleet : public FIMSObject<Type> {
       dnorm.x = fims::log(this->observed_index_data->at(i));
       dnorm.mean = fims::log(this->expected_index[i]);
       nll -= dnorm.evaluate(true);
-      fims_log::get("fleet.log")
+      FLEET_LOG
```

Comment:
line 207 should read: "observed index data" instead of "observed likelihood component"

Reply:
done